### PR TITLE
Changes in Model Construction tutorial like new screenshots,etc

### PR DESCRIPTION
**Pull request by shsa992**
Added new screenshots. Also added a content section at the top for quick navigation within this tutorial.

### CODE REVIEW

File: source/modelconstruction.rst [hunk 1]
```diff
@@ -80,21 +85,36 @@ Special to the elements in these generated mesh are node derivative parameters w
...
-Ranges of elements can be deleted from the generated mesh, but this is best done after choosing the numbers of elements options for each mesh type. Note that parts of spheres can be deleted to make e.g. bottom or top hemispheres.
+Ranges of elements can be deleted from the generated mesh, but this is best done after choosing the numbers of elements options for each mesh type. **Note** that parts of spheres can be deleted to make e.g. bottom or top hemispheres.
 
 All the current mesh types make a unit sized mesh by default, but the scale option allows this to be scaled differently in x, y and z.
 
 Task 2: Mesh Generation and Merging
 -----------------------------------
 
-Open the *DTP-ModelBuilding-Task2* workflow and execute it. Observe the mesh generation steps which make a hemisphere and a tube, then proceed to the Mesh Merger step:
+Open the *DTP-ModelConstruction-Task2* workflow and execute it. A *Mesh Generator* interface is displayed with 'Identifier: hemisphere' and other fields to input all the required values. Mention the element ID range to delete and get the bottom hemisphere as shown below:
```
Comment (hsorby) [inline source/modelconstruction.rst]: Mention the element ... this sentence doesn't work.

File: source/modelconstruction.rst [hunk 2]
```diff
@@ -80,21 +85,36 @@ Special to the elements in these generated mesh are node derivative parameters w
...
+
+.. figure:: _static/mesh-generator_part1.png
+   :align: center
+
+   Mesh Generator Part 1: Hemisphere
+
+Click on *Done*. On the new mesh generator interface, the 'identifier: tube' with other required input fields are displayed as below:
```
Comment (hsorby) [inline source/modelconstruction.rst]: Maybe explain this the other way around...

Click on the *Done* button.  The next interface is another mesh generator ...

File: source/modelconstruction.rst [hunk 3]
```diff
@@ -80,21 +85,36 @@ Special to the elements in these generated mesh are node derivative parameters w
...
+
+
+.. figure:: _static/mesh-generator_part2.png
+   :align: center
+
+   Mesh Generator Part 2: Tube
+
+After the mesh generation of the hemisphere and the tube, clicking on *Done* takes you to the 'Mesh Merger' stage. Observe the 'Identifier: hemisphere-tube' with merged *master=slave* nodes as shown here:
```
Comment (hsorby) [inline source/modelconstruction.rst]: We need to mention about the settings required and why they are required here as well.

**Review by hsorby — COMMENTED** [review]
Getting there with this section.  I think we need a bit more information for the new section on mesh merging though, see inline comments for further information.

File: source/modelconstruction.rst (hunk 3 above)
Comment (shsa992) [reply source/modelconstruction.rst]: Please may you provide an elaboration to this point.

Comment (hsorby) [reply source/modelconstruction.rst]: How many elements around are required and why they must match with the number of elements around for the hemisphere.

Comment (shsa992) [reply source/modelconstruction.rst]: Refer the "Tweaked Model Construction module". I have tried to explain this with examples. Please let me know if that is whats expected.

**Pull request merged**